packages/web/src/components/gcds-menu-link/gcds-menu-link.tsx

```diff
@@ -0,0 +1,95 @@
+import { Component, Element, Host, Prop, State, Method, h } from '@stencil/core';
+import { assignLanguage, observerConfig } from '../../utils/utils';
+
+@Component({
+  tag: 'gcds-menu-link',
+  styleUrl: 'gcds-menu-link.css',
+  shadow: true
+})
+export class GcdsMenuLink {
+  @Element() el: HTMLElement;
+
+  private linkElement: HTMLElement
+
+  /**
+   * Link href
+   */
+  @Prop({ reflect: true }) href!: string;
+
+  /**
+   * Current page flag
+   */
+  @Prop({ reflect: true }) current: boolean;
+
+  /**
+  * Language of rendered component
+  */
+  @State() lang: string;
+
+  @State() menuStyle: string;
```

Comment:
Could you add a comment for this state as well? Otherwise, it looks kinda like it belongs to the language part.